build/Linux+BSD/portable/copy-libs

```diff
@@ -135,6 +251,9 @@ building_on_CentOS_6() {
   copyLib libpng12.so.0
   copyLib libudev.so.0
   copyLib liblcms.so.1
+
+  # Needed by Gentoo 2.2:
+  copyLib libpulse.so.0
```

Comment:
Good Point!  I've changed the comment to be "needed if don't have pulseaudio installed", since this issue also affects arch-linux w/LXDE, as well as any other minimal distros that only use ALSA.